src/style.css

```diff
@@ -1 +1,89 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-weight: 500;
+  background-color: #e5e5e5;
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
use classes

Reply:
Only margin : 0 in the body tag will be ok?